libsrc/apple2/ser/a2.gs.s

```diff
@@ -314,20 +314,19 @@ SER_CLOSE:
 
 SER_OPEN:
         bit     $C082                   ; Check if this is a IIgs
```

Comment:
GitHub doesn't allow me to comment on line 308... There "the" comment is missing.